articles/openshift/oshift-ref-cluster-monitoring.md

```diff
@@ -0,0 +1,53 @@
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift integrated cluster monitoring
+toc_fullpath: Reference/oshift-ref-cluster-monitoring.md
+toc_mdlink: oshift-ref-cluster-monitoring.md
+---
+
+# OpenShift integrated cluster monitoring
+
+### Overview
+
+OpenShift 3.11 ships with a pre-configured and self-updating monitoring stack based on the Prometheus open source project. This ships with a set of alerts that notify the cluster admin of problems and a set of Grafana dashboards. The deployed Prometheus instance is responsible for monitoring and alerting on cluster and OpenShift components, it shouldn't be extended to monitor user applications. Users interested in leveraging Prometheus for application monitoring on OpenShift should consider using the operator lifecycle manager (https://github.com/operator-framework/operator-lifecycle-manager) to easily deploy a Prometheus Operator and setup new Prometheus instances to monitor and alert on their applications. Alertmanager is a cluster-global component for handling alerts generated by all Prometheus instances deployed in that cluster.
+
+### Resource
+
+We run cluster monitoring with persistent storage, this means that the metrics are stored on a persistent volume and can survive a pod being restarted or recreated. Each Alertmanager instance conumes 2Gi and each Prometheus instance consumes 50Gi. In a standard cluster with two infrastructure nodes this will mean 106Gi of storage will be used for cluster monitoring. If the persistent volumes fill up, or you notice them getting close to being full, you can follow this guide to expand the persistent volumes: https://docs.ukcloud.com/articles/openshift/oshift-how-expand-persistent-vols.html
+
+### Configuration
+
+The only Red Hat supported configuration options for the integrated monitoring stack are variables used at the time of deployment and can be seen here: https://docs.openshift.com/container-platform/3.11/install_config/prometheus_cluster_monitoring.html#configuring-openshift-cluster-monitoring
+
+Red Hat advises: Beyond those explicit configuration options, it is possible to inject additional configuration into the stack. However this is unsupported, as configuration paradigms might change across Prometheus releases, and such cases can only be handled gracefully if all configuration possibilities are controlled.
+
+### Accessing Web UIs
+
+`oc get routes -n openshift-monitoring` will give you the URLs you can use to access Prometheus, Alertmanage and Grafana UIs. You'll need to prepend `https://` to the addresses.
+
+### Grafana Dashboards
+
+If you navigate to the Grafana web UI you'll be able to access a number of pre-configured dashboards. These dashboards provide insight into the compute resource usage at cluster, node and pod levels. There are also USE Method dashboards at the cluster and node level. Click the dropdown in the top left on the home dashboard to select which dashboard you'd like to view.
```

Comment:
What's the grafana web UI address? We should be able to give a clue as to the URL here...

Reply:
The section right above this tells them how to get this from their cluster...